### DISCUSSION TITLE
How to access drives created in Beaker via FUSE?

### DISCUSSION THREAD

**Discussion by trevormeier** [beakerbrowser/beaker#1563] [General]
Hi there—how do I access my Hyperdrives on macOS via the filesystem? I've only installed Beaker so far.

**Comment by pfrazee** [comment]
Currently because of how difficult FUSE is to package with releases, you have you install the hyperdrive-daemon via NPM and set it up separately. If the daemon is running when beaker starts, it will use that daemon rather than starting its own.

Steps:

 1. Close beaker. (Make sure it's not running - by default it persists to tray.)
 2. Install hyperdrive-daemon using npm (`npm i -g hyperdrive-daemon`)
 3. [Follow the instructions on setting up FUSE](https://github.com/hypercore-protocol/hyperdrive-daemon#fuse)
 4. Keep the daemon running and then start Beaker

If you restart the computer, you'll need to restart the daemon. You'll also need to update the daemon periodically as new releases come out. (Beaker will eventually tell you if you've got an outdated daemon running for this use-case.)
